### PR TITLE
added doc

### PR DESCRIPTION
**Pull request by kgc00**
# Why
Adds a guide on how to configure and use react-native-vector-icons.

# How

Downloaded the library and used it in a sample project.  Wrote the guide based on my findings.

### CODE REVIEW

File: docs/pages/faq/react-native-vector-icon-usage.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+`react-native-vector-icons` is a popular library for adding vector icons into your app. This library bundles many SVG icon sets, such as [FontAwesome](https://fontawesome.com/), to help you get started quicker. This guide will help you setup your project to use `react-native-vector-icons` and show you how to import icons into your react-native components.
+
+## Setup
+
+### iOS
+
+iOS setup consists of two steps:
+
+- import `react-native-vector-icons` to your project via CocoaPods.
+- import the desired icon libraries to your project.
```
Comment (ide) [inline docs/pages/faq/react-native-vector-icon-usage.md:14]: These two sentences sound pretty similar. Distinguish them more clearly.

**Review by ide — CHANGES_REQUESTED** [review]
We have a page on vector icons here: https://docs.expo.io/guides/icons/. If there's information in this PR that would be useful to have on that page, we should think about whether it's better for developers for us to amend the existing page. `@expo/vector-icons` was built for better compatibility with Expo projects and we need to be careful that adding a guide on a library that does very similar things doesn't come off as confusing or conflicting advice. The high-order bit here is whether this new page should exist.

I also left more comments throughout, mostly to communicate some expectations for our docs. My comments likely aren't comprehensive but point out what I noticed in an initial reading.

File: docs/pages/faq/react-native-vector-icon-usage.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+title: Configuring and Using react-native-vector-icons
+---
+
+`react-native-vector-icons` is a popular library for adding vector icons into your app. This library bundles many SVG icon sets, such as [FontAwesome](https://fontawesome.com/), to help you get started quicker. This guide will help you setup your project to use `react-native-vector-icons` and show you how to import icons into your react-native components.
+
+## Setup
+
+### iOS
+
+iOS setup consists of two steps:
```
[the remark this answers is not in the document]
Comment (kgc00) [reply docs/pages/faq/react-native-vector-icon-usage.md:11]: For sure.  I just made a new repo and tried to skip steps.  Turns out you do not need to add anything to your podfile, but all the other steps are required.

File: docs/pages/faq/react-native-vector-icon-usage.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kgc00) [reply docs/pages/faq/react-native-vector-icon-usage.md:14]: good point, i adopted `react-native-vector-icon` terminology by referring to them as 'icon sets' in a rewrite.

**Pull request closed without merging**